staging/cse/windows/provisioningscripts/kubeletstart.ps1

```diff
@@ -31,6 +31,16 @@ if ($global:ContainerRuntime -eq "containerd") {
 
 ipmo $global:HNSModule
 
+function
+Test-NodeIPArgExists($argList) {
+    foreach ($arg in $argList) {
+        if ($arg.ToLower() -like "--node-ip*") {
```

Comment:
why the wild card?

Reply:
oh no this is just to --node-ip= 
Can we fully define to the = so we don't catch other random arguments?

Reply:
it's testing to see if `--node-ip={already provided my ip}` is in the kubelet args list already. So if for whatever reason something has already set it in the kubelet config we don't want to overwrite it (or maybe we do but I am taking the very passive approach here). Guess it could be clearer if I added the `=` in the search string

Reply:
updated with `=`